packages/xo-web/src/xo-app/home/index.js

```diff
@@ -979,7 +997,7 @@ export default class Home extends Component {
                   })}
             </span>
           </Col>
-          <Col mediumSize={8} className='text-xs-right hidden-sm-down'>
+          <Col mediumSize={6} className='text-xs-right hidden-sm-down'>
```

Comment:
Can you check if all the sizes work fine with all the widths? If not, we can take some space from the 1st column, I think.